lightkurve/seismology/core.py

```diff
@@ -602,11 +602,24 @@ def estimate_radius(self, teff, numax=None, deltanu=None):
         """
         numax = self._validate_numax(numax)
         deltanu = self._validate_deltanu(deltanu)
+        if teff == None:
```

Comment:
Nitpicking: as a general rule we have to use `is None`, because an object *could* be passed to `teff` that defines a weird `==` operator:
```suggestion
        if teff is None:
```